code/game/objects/structures/tables_racks.dm

```diff
@@ -70,7 +70,7 @@
 			if(pushed_mob.buckled)
 				to_chat(user, "<span class='warning'>[pushed_mob] is buckled to [pushed_mob.buckled]!</span>")
 				return
-			if(user.a_intent == INTENT_GRAB)
+			if(user.a_intent == INTENT_GRAB || INTENT_HARM)
```

Comment:
that wouldn't work
```suggestion
			if(user.a_intent == INTENT_GRAB || user.a_intent == INTENT_HARM)
```

Reply:
https://user-images.githubusercontent.com/53384660/150215008-e3822beb-d630-4953-b3a8-65332e682b93.mp4

It does infact work :)

Reply:
ok odd because the `||` isn't supposed to work that way lmao